Controller.cs

```diff
@@ -25,7 +25,68 @@ public static class Controller {
         public const int P16 = 16;
         public const int P19 = 19;
         public const int P20 = 20;
-        public const string BUILTIN_TEXT_LED = "led";
+
+        public const int LED = 21;
+        public const int BUZZER = 22;
+        public const int A = 23;
```

Comment:
Since DUE Lang does not have classes, if we want to be consistent with the constants it might be better to give these constants full names. Right now on the C# side you would use Controller.A or Controller.X. In DUE Lang it would be better to have BtnA or ButtonA, XAxis, YAxis etc. so to be consistent we should use the same names on the C# side.